nntrainer/layers/layer_context.h

```diff
@@ -171,7 +172,8 @@ class InitLayerContext {
   /**
    * @brief Request a new weight for the layer
    *
-   * @param dim dimension of the weight
+   * @param dim_v dimension of Variagble of the weight
```

Comment:
```suggestion
   * @param dim_v dimension of Variable of the weight
```

Also, `@param dim_v` doesn't seem to match the function prototype.

Reply:
yes. My bad. Fixed. Thanks!